application-workloads/github-enterprise/github-enterprise/azuredeploy.json

```diff
@@ -70,7 +70,7 @@
     "addressPrefix": "10.0.0.0/16",
     "subnetName": "Subnet",
     "subnetPrefix": "10.0.0.0/24",
-    "storageAccountType": "Premium_LRS",
+    "storageAccountType": "Standard_LRS",
```

Comment:
is the storageAccount even used?

Reply:
GitHub Actions requires a storage account, I imagine this is why it was included. If GitHub Actions are not being enabled then the storage account is not required/utilised. 

Removal of the storage account completely from the template is certainly a solution. Removal will save those who aren't using Actions from extra storage account charges. But if it is to stay then it needs to be changed as the current setting is incompatible with GitHub Actions. 

The official documentation for [Enabling GitHub Actions](https://docs.github.com/en/enterprise-server@3.4/admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-azure-blob-storage) describes setting up a storage account as the first step so removal from the template seems appropriate.

Reply:
If the expectation is that GH Actions will be enabled as a later/separate step - then it makes sense to remove it.  If Actions are enabled as part of the template, we can leave it - but then should should probably add a comment as to how it will be used, since it's never referenced in the template.  That would be my vote...

Reply:
Hi @bmoore-msft, after some discussions internally I'd like to request we update the storage account to a compatible one and leave it in place. 

This will allow customers to configure actions on first setup without needing to configure external storage separately. If Actions is not used, the the storage account will still be created but not used. I was under the impression this would involve unneeded cost for the user but this is not the case.

Can we please proceed with the initial change request to update this storage account line to prevent issues for new customers enabling actions. 

Thanks,